### DISCUSSION TITLE
Cannot get started using strapi

### DISCUSSION THREAD

**Discussion by arifikhsan** [strapi/strapi#6255] [Legacy Topics]
**Describe the bug**
I cannot get started with strapi. the installation gives me error

**Steps to reproduce the behavior**

1. Using get started command in documentation

both 
yarn create strapi-app my-project --quickstart
and
npx create-strapi-app my-project --quickstart

did not work

2. See error

**Expected behavior**
Can create new project using strapi

**Screenshots**
i dont know whats happen

![image](https://user-images.githubusercontent.com/32485694/82247224-3e5eab80-9970-11ea-8625-233031395cbc.png)

![image](https://user-images.githubusercontent.com/32485694/82247297-5e8e6a80-9970-11ea-9c3c-6a039ba58786.png)

**System**

- Node.js version: 12.14.1
- NPM version: 6.13.4
- Strapi version: latest
- Database: -- quick install
- Operating system: Windows 10

**Comment by robertpiosik** [comment]
Hey, please google `not recognized as an internal or external command` and try the solutions. It's your system specific error.

**Comment by derrickmehaffy** [comment]
I also would not recommend cmd as an admin, but it appears you have some permission issues. Going to move this to a discussion as it's not directly a bug with Strapi but is a user environment issue.
